@here/harp-mapview/lib/ClipPlanesEvaluator.ts

```diff
@@ -721,49 +644,33 @@ export class TiltViewClipPlanesEvaluator extends TopViewClipPlanesEvaluator {
         // intersection distance determines the near distance.
         const planesDist = this.getFrustumGroundIntersectionDist(camera, projection);
 
-        if (camera instanceof THREE.PerspectiveCamera) {
-            // Project intersection distances onto the eye vector.
-            // Angle between top/bottom plane and eye vector is half of the vertical fov.
-            const halfFovAngle = THREE.MathUtils.degToRad(camera.fov / 2);
-            const cosHalfFov = Math.cos(halfFovAngle);
-            // cos(halfFov) = near / bottomDist
-            // near = cos(halfFov) * bottomDist
-            viewRanges.near = planesDist.bottom * cosHalfFov;
-            // cos(halfFov) = far / topDist
-            // far = cos(halfFov) * topDist
-            viewRanges.far = planesDist.top * cosHalfFov;
-        } else {
-            // No projection needed for orthographic camera, clip planes are parallel to eye vector.
-            assert(camera instanceof THREE.OrthographicCamera, "Unsupported camera type.");
-            viewRanges.near = planesDist.bottom;
-            viewRanges.far = planesDist.top;
-        }
-
+        // Project intersection distances onto the eye vector.
+        // Angle between top/bottom plane and eye vector is half of the vertical fov.
+        const halfFovAngle = THREE.MathUtils.degToRad(camera.fov / 2);
```

Comment:
above you just have `halfFov`, so best rename this too

Reply:
done